srcpkgs/TMSU/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'TMSU'
+pkgname=TMSU
+version=0.7.4
+revision=1
+wrksrc="${pkgname}-${version}"
+create_wrksrc=yes
+build_style=gnu-makefile
+make_cmd="make -C ${wrksrc} MOUNT_INSTALL_DIR=/usr/bin"
+hostmakedepends="go git"
+depends="fuse sqlite>=3"
+short_desc="Tool for tagging files and accessing them through FUSE"
+maintainer="xaltsc <xaltsc@protonmail.ch>"
+license="GPL-3.0-or-later"
+homepage="https://tmsu.org"
+distfiles="https://github.com/oniony/${pkgname}/archive/v${version}.tar.gz"
+checksum=04e8a4c47532a4d8f673c3b3a921a47815c752b2ca1b564f6e1ed58a1b88a177
+nopie=yes
```

Comment:
no, make it respect our CFLAGS and build proper pie binaries.

Reply:
I changed almost everything trying to build pie binaries.